client/blockchain-service/src/handler.rs

```diff
@@ -525,6 +535,22 @@ impl Actor for BlockchainService {
                         }
                     }
                 }
+                BlockchainServiceCommand::QueryStorageProviderId { account, callback } => {
```

Comment:
@snowmead also added a similar functionality to this one. So depending on who merges first, try to make it so that they don't collide.